apps/spruce/src/pages/projectSettings/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -196,6 +201,10 @@ export const getFormSchema = (
         mergeQueue: {
           type: "object" as "object",
           title: "Merge Queue",
+          ...(projectType === ProjectType.Repo && {
+            description:
+              "If enabled, these settings can only apply to one branch project that also has this feature enabled. This does not apply to untracked branches.",
```

Comment:
Confirmed that this worked
<img width="814" alt="image" src="https://github.com/user-attachments/assets/c1b093ab-e644-4b74-930b-ef18eab0777d" />